<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Исходный текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Зашифрованный текст</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_original">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_encrypted">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_encrypt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Зашифровать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_decipher">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Расшифровать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Открытый ключ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_key">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сообщения о работе программы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_message">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_generate_key">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>331</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сгенерировать ключ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
